cmd/upgrade/machinepool/cmd.go

```diff
@@ -102,69 +124,194 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command, argv []string) error {
 	clusterKey := r.GetClusterKey()
 	cluster := r.FetchCluster()
 	machinePoolID := argv[0]
-	scheduleDate := args.scheduleDate
-	scheduleTime := args.scheduleTime
+	var err error
+	currentUpgradeScheduling := ocm.UpgradeScheduling{
+		Schedule:                 args.schedule,
+		ScheduleDate:             args.scheduleDate,
+		ScheduleTime:             args.scheduleTime,
+		AllowMinorVersionUpdates: args.allowMinorVersionUpdates,
+		AutomaticUpgrades:        args.schedule != "",
```

Comment:
Should the "AutomaticUpgrade" flag have also the or args.schedule != "" || (args.scheduleDate != "" && args.scheduleTime != "") here ?

Reply:
automatic upgrade requires a schedule in cron format, that is passed with `schedule` paramater. 
manual schedule requires a `scheduleDate` and `scheduleTime` because it is a single schedule.
So here we say that enabling automatic upgrades is equivalent to specify `schedule`. So it is correct.